tests/integration_tests/test_integration_dds.py

```diff
@@ -341,6 +341,29 @@ def test_exclude_runfolder_from_batch_delivery(self):
                 status_response = yield self.http_client.fetch(link)
                 self.assertEqual(json.loads(status_response.body)["status"], StagingStatus.staging_successful.name)
 
+    @gen_test()
+    def test_exclude_allrunfolders_from_batch_delivery(self):
+        """Should raise an exception since all runfolders are excluded"""
+        with tempfile.TemporaryDirectory(dir='./tests/resources/runfolders/',
+                                         prefix='160930_ST-E00216_0555_BH37CWALXX_') as tmpdir1, \
+                tempfile.TemporaryDirectory(dir='./tests/resources/runfolders/',
+                                            prefix='160930_ST-E00216_0556_BH37CWALXX_') as tmpdir2:
+            self._create_projects_dir_with_random_data(tmpdir1, 'XYZ_123', os.path.basename(tmpdir1))
+            self._create_projects_dir_with_random_data(tmpdir2, 'XYZ_123', os.path.basename(tmpdir2))
+            
+            url = "/".join([self.API_BASE, "stage", "project", 'runfolders', 'XYZ_123'])
+            exclude_runfolder = [os.path.basename(tmpdir1), os.path.basename(tmpdir2)]
+            payload = {'delivery_mode': 'FORCE', 'exclude': exclude_runfolder}
+            from delivery.exceptions import ProjectNotFoundException
+            with self.assertRaises(Exception) as e:
```

Comment:
Shouldn't `ProjectNotFoundException` be used here instead of a general Exception?